### PR TITLE
Js objectsAwesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by Charzy-1**
I built a basic website that allows users to add/remove books from a list. 
javaScript objects and arrays was used to achieve my task.
I created three functions for this purpose in js. one of the function displays the books, the second add new books and their corresponding authors while the last removes the right book
I dynamically modified the DOM and added basic events.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (Pamphilemkp) [inline .github/workflows/linters.yml]: While there are Webhint and stylelints jobs running, there is a missing job for ESLINT kindly add it the file.
## Hint 
check  for the corresponding linters in the linter's list:
 HTML,CSS and JS and update this file.

**Review by Pamphilemkp — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @Charzy-1 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## Highlights :green_circle: 

- user can add and remove books
- All the books are displayed to the UI 
- PR is descriptive and professional :+1: 

## Required Changes ♻️



_Check the comments under the review._

## Optional suggestions



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy] you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form]

**Review by Pamphilemkp — APPROVED** [review]
Hi @Charzy-1 ,

## APPROVED 🟢


## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy] you have a limited number of reviews per project (check the exact number in your Dashboard).

**Comment by Charzy-1** [comment]
Thank you

**Pull request merged**